tests/baselines/reference/isolated-declarations/auto-fixed/diff/declarationEmitBindingPatterns.d.ts.diff

```diff
@@ -0,0 +1,17 @@
+// [[Reason: #56992 Type printer preserves binding element aliases.]] ////
+
+//// [tests/cases/compiler/declarationEmitBindingPatterns.ts] ////
+
+===================================================================
+--- TSC	declarations
++++ DTE	declarations
+@@ -1,8 +1,8 @@
+ 
+ 
+ //// [declarationEmitBindingPatterns.d.ts]
+-declare const k: ({ x: z }: {
++declare const k: ({ x }: {

```

Comment:
Similarly here; I would have thought this would go away.

Reply:
Yes. Now it's just a source map difference